cmd/evmosd/testnet.go

```diff
@@ -148,6 +154,18 @@ Example:
 			args.startingIPAddress, _ = cmd.Flags().GetString(flagStartingIPAddress)
 			args.numValidators, _ = cmd.Flags().GetInt(flagNumValidators)
 			args.algo, _ = cmd.Flags().GetString(flags.FlagKeyType)
+			baseFee, _ := cmd.Flags().GetString(flagBaseFee)
```

Comment:
ditto here, this should be Uint
```suggestion
			baseFee, _ := cmd.Flags().GetUint64(flagBaseFee)
```

Reply:
Uint64 is not enough to express a large baseFee, string is required